solutionbox/inception/datalab_solutions/inception/_local.py

```diff
@@ -46,19 +49,21 @@ def preprocess(self, input_csvs, labels_file, output_dir):
                                    output_dir, job_id)
     p.run()
 
-  def train(self, labels_file, input_dir, batch_size, max_steps, output_path):
+  def train(self, labels_file, input_dir, batch_size, max_steps, output_dir):
     """Local training."""
 
     num_classes = len(_util.get_labels(labels_file))
     model = _model.Model(num_classes, 0.5, self._checkpoint)
     task_data = {'type': 'master', 'index': 0}
     task = type('TaskSpec', (object,), task_data)
-    _trainer.Trainer(input_dir, batch_size, max_steps, output_path,
+    _trainer.Trainer(input_dir, batch_size, max_steps, output_dir,
```

Comment:
side question: is there anything wrong with a package author making a subprocess call to gcloud beta local train?